rpcserver.go

```diff
@@ -2728,11 +2727,68 @@ func (r *rpcServer) GetNetworkInfo(ctx context.Context,
 		minChannelSize = 0
 	}
 
-	// TODO(roasbeef): graph diameter
+	// Calculate graph diameter: Diameter = maximum length of shortest path
+	// in graph between two nodes.
+	var diam int
+
+	// We'll run through all the nodes in graph, find farthest node from
+	// each, and take the maximum of that.
+	if err := graph.ForEachNode(nil, func(tx *bolt.Tx, SourceNode *channeldb.LightningNode) error {
+
+		// This will store the farthest node
+		var maxdist = -1
```

Comment:
As the variable is being initialized, and declared at the same time, we can simply use the following syntax: 
```
maxDist := -1 
```

Also we use camel case (lower camel) as style within the project.